src/main/java/frc/robot/subsystems/GamePieceManipulatorSubsystem.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.subsystems;
+
+import static frc.robot.Constants.GamePieceManipulatorConstants.DEVICE_ID_MANIPULATORMOTOR;
+import static frc.robot.Constants.GamePieceManipulatorConstants.INTAKESPEED;
+import static frc.robot.Constants.GamePieceManipulatorConstants.OUTTAKESPEED;
+import static frc.robot.Constants.GamePieceManipulatorConstants.SCORESPEED;
+
+import com.ctre.phoenix6.controls.VelocityTorqueCurrentFOC;
+import com.ctre.phoenix6.hardware.TalonFX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+/**
+ * The is the Subsytem for the Game Pieace Manipulator.
+ */
+public class GamePieceManipulatorSubsystem extends SubsystemBase {
+  /** Creates a new Subsytem for the Game Pieace Manipulator. */
+
+  // define motors
+  private TalonFX manipulatorMotor = new TalonFX(DEVICE_ID_MANIPULATORMOTOR);
```

Comment:
All of the motors will be connected to the CANivore, so we need to include the CAN bus ID in the call to the constructor